app/views/sp_components/new.html.erb

```diff
@@ -1,22 +1,22 @@
 <fieldset class="govuk-fieldset">
   <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
-    <h1 class="govuk-fieldset__heading">Upload your SP component</h1>
+    <h1 class="govuk-fieldset__heading"><%= t 'components.upload_sp' %></h1>
   </legend>
 
   <%= error_messages_for(@component&.errors)%>
   <%= form_for @component, url: sp_components_path, as: :component do |f| %>
     <div class="govuk-form-group <%= 'govuk-form-group--error' if @component.errors.key?(:name) %>">
-      <h3 class="govuk-heading-m">SP Component name</h3>
+      <h3 class="govuk-heading-m"><%= t 'components.sp_component_name' %></h3>
       <%=error_message_on(f.object.errors, :name) %>
       <%= f.text_field :name, class: "govuk-input#{@component.errors.key?(:name) ? ' govuk-input--error' : ''}" %>
     </div>
 
     <div class="govuk-form-group <%= 'govuk-form-group--error' if @component.errors.key?(:component_type) %>">
-      <h3 class="govuk-heading-m">SP Component type</h3>
+      <h3 class="govuk-heading-m"><%= t 'components.sp_component_type' %></h3>
       <%=error_message_on(f.object.errors, :component_type) %>
       <div class="govuk-radios govuk-radios--inline">
         <div class="govuk-radios__item ">
-          <%= f.radio_button :component_type, COMPONENT_TYPE::VSP, class: 'govuk-radios__input'  %>
+          <%= f.radio_button :component_type, COMPONENT_TYPE::VSP, class: 'govuk-radios__input'  %> <!-- enums? -->
```

Comment:
What happens to enums with localization?

Reply:
this isn't an enum. The constant is providing the correct string, so we can leave this as it is. With more content we might actually use the constants to pick the right locale string.